packages/common/util.js

```diff
@@ -31,10 +31,13 @@ const log = require('./log');
  *
  * @alias module:util
  */
+const warned = new Set();
```

Comment:
You might want to check the generated docs and make sure that the JSDoc here isn't now documenting the `const warned ...` line.